.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
```

Comment:
Do we really want to include IDE specific files now?

Reply:
@alexplischke my bad, I took it out.